pkg/timelock/const_test.go

```diff
@@ -2,19 +2,19 @@ package timelock
 
 import (
 	"math/big"
-	"os"
 
-	"github.com/smartcontractkit/timelock-worker/pkg/logger"
+	"github.com/rs/zerolog"
+	"github.com/samber/lo"
 )
 
 var (
-	testNodeURL                 = os.Getenv("TEST_NODE_URL")
-	testTimelockAddress         = os.Getenv("TEST_TIMELOCK_ADDRESS")
-	testCallProxyAddress        = os.Getenv("TEST_PROXY_ADDRESS")
-	testPrivateKey              = os.Getenv("TEST_PRIVATE_KEY")
+	testNodeURL                 = "ws://node.url"
```

Comment:
we no longer depend on pre-deployed test contracts whose info is unknown to anyone because it's stored as secrets on github

Reply:
Since you are removing env variables here, are [these](https://github.com/smartcontractkit/timelock-worker/blob/91b76004d5808d343f75b6ff6de5a3b4a0d1978a/.github/workflows/push-main.yml#L37) env parameters still needed in GH workflows?

Reply:
fixed in the latest commit.